web-components/tsconfig.json

```diff
@@ -5,22 +5,17 @@
     "declaration": true,
     "jsx": "react-jsx",
     "noEmitOnError": true,
+    "types": ["node", "jest"],
     "outDir": "lib",
     "rootDir": "src",
     "module": "esnext",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
     "isolatedModules": true,
-    "noEmit": true
```

Comment:
this part is important: if `noEmit` is set to `true`, our TS files will never be compiled into `/lib` and wont’ be usable in the registry app